colorPos2d[6] color_pos_2d_list
